ui/raidboss/data/triggers/shinryu-ex.js

```diff
@@ -33,115 +37,197 @@
       regex: /:Shinryu starts using Akh Morn on (\y{Name})/,
       alertText: function(data, matches) {
         if (matches[1] == data.me)
-          return 'Akh Morn on you';
+          return {
+            en: 'Akh Morn on you',
+            de: 'Akh Morn auf dir',
+          };
         if (data.role == 'tank')
-          return 'Akh Morn on ' + matches[1];
+          return {
+            en: 'Akh Morn on ' + matches[1],
+            de: 'Akh Morn auf ' + matches[1],
+          };
       },
       infoText: function(data, matches) {
         if (matches[1] == data.me || data.role == 'tank')
           return;
-        return 'Akh Rhai: spread and move';
+        return {
+          en: 'Akh Rhai: spread and move',
+          de: 'Akh Rhai: Verteilen und bewegen',
+        };
       },
       tts: 'akh morn',
     },
     { id: 'ShinryuEx Diamond Dust',
       regex: /:25DD:Shinryu starts using Diamond Dust/,
-      infoText: function(data) { return 'Ice: Stack and Stop'; },
+      regexDe: /:25DD:Shinryu starts using Diamantenstaub/,
+      infoText: function(data) { return {
+        en: 'Ice: Stack and Stop',
+        de: 'Eis: Stack und Stehenbleiben',
+      }; },
       tts: 'stop',
     },
     { id: 'ShinryuEx Dragonfist',
       regex: /:Shinryu starts using Dragonfist/,
-      infoText: function(data) { return 'Out of middle'; },
+      regexDe: /:Shinryu starts using Drachenfaust/,
+      infoText: function(data) { return {
+        en: 'Out of middle',
+        de: 'Raus aus der Mitte',
+      }; },
       tts: 'out of middle',
     },
     { id: 'ShinryuEx Hellfire',
       regex: /:25DB:Shinryu starts using Hellfire/,
+      regexDe: /:25DB:Shinryu starts using Höllenfeuer/,
       durationSeconds: 7,
-      alertText: function(data) { return 'Get in water'; },
+      alertText: function(data) { return {
+        en: 'Get in water',
+        de: "In's Wasser",
+      }; },
       tts: 'water',
     },
     { id: 'ShinryuEx Hypernova',
       regex: /:Right Wing starts using Hypernova/,
+      regexDe: /:Rechter Flügel starts using Supernova/,
       durationSeconds: 7,
       alertText: function(data) {
         if (data.phase == 3)
```

Comment:
This is just a personal style thing, but any block that's more than one line long should get curly braces around it.